packages/core/core-flows/src/fulfillment/steps/upsert-shipping-options.ts

```diff
@@ -65,7 +61,13 @@ export const upsertShippingOptionsStep = createStep(
 
     const upsertedShippingOptions: ShippingOptionDTO[] =
       await fulfillmentService.upsertShippingOptions(
-        input as UpsertShippingOptionDTO[]
+        input.map(i => {
```

Comment:
suggestion: I am not super fan of one letter naming for multi line function (I do accept it for oneliner thouhg ;) )